daemon/volumes_unix.go

```diff
@@ -38,7 +38,7 @@ func copyOwnership(source, destination string) error {
 // setupMounts iterates through each of the mount points for a container and
 // calls Setup() on each. It also looks to see if is a network mount such as
 // /etc/resolv.conf, and if it is not, appends it to the array of mounts.
-func (container *Container) setupMounts() ([]execdriver.Mount, error) {
+func (container *Container) setupMounts(includeIpcMounts bool) ([]execdriver.Mount, error) {
```

Comment:
Can we do this without the bool argument?
I don't think we need to worry about it since it's possible to copy this today via the API, which is what `mountVolumes` is used for.